tests/integration/ha_tests/manifests/extend_pebble_restart_delay.yml

```diff
@@ -0,0 +1,5 @@
+services:
+  mongod:
+    override: merge
+    backoff-delay: 150s
+    backoff-limit: 150s
```

Comment:
**niptick** shouldn't this be 180s to convert to 3min, as mentioned in the comments?